.github/workflows/codeql.yml

```diff
@@ -30,10 +30,10 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1

```

Comment:
thanks for changing the `init`
could you also update the `analyze`? 😋 
https://github.com/github/codeql-action#usage